server/public/styles.css

```diff
@@ -115,6 +118,33 @@ ul {
   list-style-type: none;
   padding: 0.7rem 15px;
   margin-bottom: 10px;
+  position: relative;
+}
+
+.center {
+  text-align: center;
+}
+
+.fa-arrow-right {
+  position: absolute;
+  right: 10px;
+  top: 25px;
+  font-size: 1.5rem;
+}
+
+.location-page {
+  padding: 10px;
+}
+
+.lp {
+  top: 8px;
+}
+
+.fa-arrow-left {
```

Comment:
Instead of styling classes from libraries, you should create a custom class to use next to it.